src/midiEditorCommands.cpp

```diff
@@ -661,3 +658,188 @@ void cmdMidiFilterWindow(Command *command) {
 }
 
 #endif // _WIN32
+
+void postMidiChangeVelocity(int command) {
+	if (!shouldReportNotes) {
+		return;
+	}
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);	
+	// Get selected notes.
+	vector<MidiNote> selectedNotes = getSelectedNotes(take);
+	if (!selectedNotes.size()) {
```

Comment:
nit: Because we're talking about size, I think `== 0` is more readable.
```suggestion
	if (selectedNotes.size() == 0) {
```